build-packages.sh

```diff
@@ -20,10 +29,14 @@ build_dir_is_usable () {
     echo "invalid build directory, no path allowed: \"$build\""
     return 1
   fi
-  git ls-files --error-unmatch "$build" &> /dev/null
-  if [ $? == 0 ]; then
-    echo "invalid path, \"$build\" is under revision control"
-    return 1
+  if [[ -d ".git" || $BUILD_PROHIBIT_GIT -eq 1 ]]; then
```

Comment:
I would prefer if the ".git" verification was done in the main body of the script, at the beginning and, based on that, we set a variable ourselves to mean if git should be used or not.

Reply:
Fixed.